.github/workflows/tests.yaml

```diff
@@ -18,9 +18,9 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - name: Install dependencies
       run: |
-        python -m pip install --upgrade pip
-        pip install -r requirements.txt
-        pip install black isort
+        python -m pip install --upgrade pip pipenv pipfile
+        pipenv install --system --deploy --ignore-pipfile
```

Comment:
Using pipenv here is a bit unnecessary, but when we use it, why not install dev packages wiht `--dev`.